ranger-plugin/scripts/enable-ranger-plugin.sh

```diff
@@ -20,7 +20,7 @@
 #
 
 function usage() {
-  echo "USAGE: enable-ranger-plugin.sh -r ranger_host:ranger_port -u ranger_user -p ranger_password [-h hawq_host:hawq_port] -w hawq_user -q hawq_password"
+  echo "USAGE: enable-ranger-plugin.sh -r ranger_host:ranger_port -u ranger_user -p ranger_password [-h hawq_host:hawq_port -c hawq_kerberos_service_name] -w hawq_user -q hawq_password -t lookup_authentication_type"
```

Comment:
since lookup_authentication_type will determine whether to have option "-c hawq_kerberos_service_name", I think it would be better to place option "lookup_authentication_type" before "-c hawq_kerberos_service_name".